docs/speech/index.rst

```diff
@@ -55,14 +54,8 @@ See: `Speech Asynchronous Recognize`_
     ...         sample_rate_hertz=44100,
     ...     ),
     ... )
-    >>> retry_count = 100
-    >>> while retry_count > 0 and not operation.complete:
-    ...     retry_count -= 1
-    ...     time.sleep(10)
-    ...     operation.poll()  # API call
-    >>> operation.complete
-    True
-    >>> for result in operation.results:
+    >>> opresult = operation.result()
```

Comment:
The "word boundary" should be evident in variable names, so here it'd be `op_result`.

Reply:
fixed.

Reply:
i'll upload the fix when i fix the mock issue.